apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -71,6 +71,21 @@
          */
         public static boolean IS_OPEN_DEBUGGING_CLASS = false;
 
+        /**
+         * Specify register.status dir ,This is an option, the default is AGENT_HOME/option/reset.status.
+         */
+        public static String REGISTER_STATUS_DIR = "skywalking-agent/option";
+
+        /**
+         * Specify instance_uuid to ensure that the whole show is unique, for example: applicationName_ip_12
+         */
+        public static String INSTANCE_UUID = "";
```

Comment:
I haven't found any place using this `INSTANCE_UUID`. I remember the design doc said, it should replace `ServiceAndEndpointRegisterClient#PROCESS_UUID` if this field is not empty.

Reply:
this INSTANCE_UUID has be used